src/ui/AdvancedSearch/KeywordsInput/AllKeywordsInput.ts

```diff
@@ -5,4 +5,12 @@ export class AllKeywordsInput extends KeywordsInput {
   constructor(public root: HTMLElement) {
     super(l('AllTheseWords'), root);
   }
+
+  public getValue() {
+    const value = super.getValue();
+    if (!value) {
+      return '';
+    }
+    return `<@-${value}-@>`;
```

Comment:
The biggest possible error would be if `value` happened to contain `-@>`, but I don't believe that's a big concern.